Tests/test_image_getdata.py

```diff
@@ -14,7 +14,7 @@ def test_sanity() -> None:
     assert data[0] == (20, 20, 70)
 
 
-def test_roundtrip() -> None:
+def test_hopper() -> None:
```

Comment:
```suggestion
def test_mode() -> None:
```
How about this instead?